### DISCUSSION TITLE
Skeleton Documentation

### DISCUSSION THREAD

**Discussion by shiroze** [react-native-elements/react-native-elements#3662] [Q&A]
I'm using skeleton to for content loader from this documentation but it show me black bar and blinking

Source : https://reactnativeelements.com/docs/components/skeleton

Screenshot : 
![image](https://user-images.githubusercontent.com/6022114/197668305-8d1e4694-1d58-4825-965e-3d27cc1e8599.png)

My Testing Code : 

```
<View style={styles.container}>
      <Text>Your View Item</Text>
      <Skeleton />
</View>
```

What I want to achieve, it show same color like the screenshot below

![image](https://user-images.githubusercontent.com/6022114/198844611-077be6e9-3d66-4a2e-991d-1849c179af5c.png)

**Comment by arpitBhalla** [comment]
Try specifying `height`?

**Reply by shiroze** [reply]
Same, I was using default animation, but if I change into wave it show wave animation with black color moving from left to right.
Can we use like background color / color ? where can I put this color things ? did I must specify LinearGradient ?
